instrumentation/jsonrpc4j-1.3/library/src/main/java/io/opentelemetry/instrumentation/jsonrpc4j/v1_3/JsonRpcClientAttributesGetter.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.jsonrpc4j.v1_3;
+
+import io.opentelemetry.instrumentation.api.incubator.semconv.rpc.RpcAttributesGetter;
+
+// Check
+// https://github.com/open-telemetry/semantic-conventions/blob/main/docs/rpc/rpc-metrics.md#attributes
+// Check https://opentelemetry.io/docs/specs/semconv/rpc/json-rpc/
+public enum JsonRpcClientAttributesGetter implements RpcAttributesGetter<JsonRpcClientRequest> {
```

Comment:
does this class need to be public? if so move it to an `internal` package

Reply:
removed `public`